flink-runtime/src/main/java/org/apache/flink/runtime/rest/util/RestMapperUtils.java

```diff
@@ -33,10 +40,23 @@
 		objectMapper.enable(
 			DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES,
 			DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES,
-			DeserializationFeature.FAIL_ON_READING_DUP_TREE_KEY,
-			DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES);
```

Comment:
I had to remove `FAIL_ON_MISSING_CREATOR_PROPERTIES` because `null` fields are not always represented in the JSON. The `RestClient` would otherwise run into problems.